src/webview/js/js.js

```diff
@@ -491,18 +502,3 @@ documentBody.addEventListener('touchmove', (e: TouchEvent) => {
 documentBody.addEventListener('drag', (e: DragEvent) => {
   lastTouchEventTimestamp = 0;
 });
-
-/*
- *
- * Synchronizing setup with outside
- *
- */
-
-const waitForBridge = () => {
```

Comment:
Now, you are assuming the WebView will be immediately ready to send messages.
That might be true in Chrome 70 but not in previous (or even future) versions!

Reply:
> you are assuming the WebView will be immediately ready to send messages.

kind of.

If it fails then React component will again send `hello` event and at that time it will respond.

Anyway we need to trust one side. So I thinks it better to have complexity on that side where we can debug easily, and make less debugging side with less logic.